### ISSUE TITLE
Create objects but only (c)rudl these created ones and all related

### ISSUE THREAD

**Issue by IamSAB** [trilbymedia/grav-plugin-flex-objects#147]
I just read the flex objects docs on grav. Currently I try to find out if the following usecase is achievable:

Is it possible to set permissions on a collection, so that a registered user can create objects, but only read, update and delete created ones. In addition, he should be able to do the same for all objects which are related to the objects he created.

My plan is to port this idea to Grav using flex-objects:

- sync contacts via caldav from addressbook (using custom storage and formatter)
- let families register and update their contacts (parents and children)

I would like to know, how much effort it takes, what are the most important things to consider and main steps I should take. The docs are good but don't show how and if the above idea is realizable. This question is to get the missings parts of the docs concerning my use case.

Thank you

**Comment by mahagr** [comment]
Yes, it's already working on pages, but for the other objects, you need to add logic to support this.
